oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/AlarmQuery.java

```diff
@@ -62,11 +85,166 @@ public Alarms getAlarm(final Duration duration, final Scope scope, final String
         }
         long startSecondTB = 0;
         long endSecondTB = 0;
+        EventQueryCondition condition = new EventQueryCondition();
         if (nonNull(duration)) {
             startSecondTB = duration.getStartTimeBucketInSec();
             endSecondTB = duration.getEndTimeBucketInSec();
+            condition.setTime(duration);
+        }
+        condition.setType(EventType.Error);
+        condition.setName("Alarm");
```

Comment:
These conditions are specific to alarm events, which is not what we want.

Let's take an example, if there is an alarm saying that `cpm(serviceA_Instance1) == 0`, we should include all events of `serviceA_Instance1` during that period, they may be `Shutdown` event, `Crash` event, etc. so that users can make sense of the alarm from the events quickly

Reply:
Yeah, which means that there are fewer filters to query Events, which means that when an alert occurs, the number of Events associated is no longer the corresponding event, but all related Events.

What I wanted to establish was whether Alarms and Event(s) had a unique connection, rather than all of the events associated with the application in a given time period.

Because this association method may not be accurate and accurate enough in special scenarios, it may produce incorrect association relationship.What do you think? @kezhenxu94  @wu-sheng .

Looking forward to your reply. Thanks a lot.